inttest/common/footloosesuite.go

```diff
@@ -837,24 +838,55 @@ func (s *FootlooseSuite) initializeFootlooseClusterInDir(dir string) error {
 	}
 
 	volumes := []config.Volume{
-		{
-			Type:        "bind",
-			Source:      "/lib/modules",
-			Destination: "/lib/modules",
-		},
 		{
 			Type:        "bind",
 			Source:      binPath,
 			Destination: s.K0sFullPath,
+			ReadOnly:    true,
 		},
 		{
 			Type:        "volume",
 			Destination: "/var/lib/k0s",
 		},
 	}
 
+	// Ensure that kernel config is available in the footloose boxes.
+	// See https://github.com/kubernetes/system-validators/blob/v1.6.0/validators/kernel_validator.go#L180-L190
+
+	bindPaths := []string{
+		"/usr/src/linux/.config",
+		"/usr/lib/modules",
+		"/lib/modules",
+	}
+
+	if kernelVersion, err := (&system.KernelValidatorHelperImpl{}).GetKernelReleaseVersion(); err == nil {
+		bindPaths = append(bindPaths, []string{
```

Comment:
`/proc` is treated specifically by Docker as far as I can tell. If `/proc/config.gz` is present on the host, it will be present inside the container, as well. At least that's the behavior on my box.

Reply:
ah... of course...